### ISSUE TITLE
Add Nepali Language support for the app

### ISSUE THREAD

**Issue by rajeshpanjiyar** [Swati4star/Images-to-PDF#1032]
Adding support of Nepali language in the app can be very helpful for users who are comfortable in Nepali or are habituated with that language.

*Being a native **Nepali language** speaker with Nepali as one of the subjects in academics so I would like to work on this issue*

**Comment by codegsaini** [comment]
@rajeshpanjiyar You can work on it. Your efforts will be appreciated.

**Comment by amaan118921** [comment]
i would like to take this up as a hacktoberfest contribution, please assing me this.

**Comment by amaan118921** [comment]
Hey can i take this up @codegsaini

**Comment by codegsaini** [comment]
@amaan118921 Yes, of course, you can contribute. Look, an assignment is just an indicator of the occupancy of a person on that issue, just to tell the world that someone is already working on it thus preventing different people to work on the same issue to prevent conflict. The absence of an assigned tag never restricts you to contribute.
